multicluster/controllers/multicluster/common/types.go

```diff
@@ -30,8 +30,12 @@ const (
 	InvalidClusterID    = ClusterID("invalid")
 	InvalidClusterSetID = ClusterSetID("invalid")
 
-	DefaultWorkerCount = 5
+	DefaultWorkerCount       = 5
+	LabelIdentityWorkerCount = 50
 
 	EndpointIPTypeClusterIP = "ClusterIP"
 	EndpointIPTypePodIP     = "PodIP"
+
+	ResourceExchangeQPS   = 200
+	ResourceExchangeBurst = 400
```

Comment:
ditto, maybe a few background info about why we increase these configs?

Reply:
Done.